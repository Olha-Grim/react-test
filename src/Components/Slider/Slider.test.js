import React from "react";
import { render } from "@testing-library/react";
import Slider from ".";

describe("Slider", () => {
  test("interaction with Slider component", () => {
    const { container } = render(<Slider defaultValue={30} disabled={false} />);
    expect(container.firstChild).toBeInTheDocument();
  });
});
