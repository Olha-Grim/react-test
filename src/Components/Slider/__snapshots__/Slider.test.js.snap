// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton interaction with Skeleton component 1`] = `
<DocumentFragment>
  <div
    class="ant-slider"
  >
    <div
      class="ant-slider-rail"
    />
    <div
      class="ant-slider-track"
      style="left: 0%; width: 0%;"
    />
    <div
      class="ant-slider-step"
    />
    <div
      aria-disabled="false"
      aria-valuemax="100"
      aria-valuemin="0"
      aria-valuenow="0"
      class="ant-slider-handle"
      role="slider"
      style="left: 0%; transform: translateX(-50%);"
      tabindex="0"
    />
    <div
      class="ant-slider-mark"
    />
  </div>
</DocumentFragment>
`;

exports[`Slider interaction with Slider component 1`] = `
<div
  class="ant-slider"
>
  <div
    class="ant-slider-rail"
  />
  <div
    class="ant-slider-track"
    style="left: 0%; width: 0%;"
  />
  <div
    class="ant-slider-step"
  />
  <div
    aria-disabled="false"
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="0"
    class="ant-slider-handle"
    role="slider"
    style="left: 0%; transform: translateX(-50%);"
    tabindex="0"
  />
  <div
    class="ant-slider-mark"
  />
</div>
`;
